docs/tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "skipLibCheck": true,
     "esModuleInterop": true,
     "lib": ["ESNext", "DOM"],
+    "types": ["vite-plugin-pwa/client"],
```

Comment:
What do you think about [referencing](https://github.com/userquin/iles-pwa/blob/userquin/feat-pwa-module/docs/index.d.ts#L1) the types?

Sometimes TypeScript will ignore any automatically detected types (such as `@types/node`) when specifying it manually.